modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/RosettaError.scala

```diff
@@ -35,6 +35,13 @@ object RosettaError extends IntEnum[RosettaError] with RosettaErrorEncoder {
       extends RosettaError(value = 3, message = "Invalid request body", retriable = false, details = RosettaErrorDetails(reason).some)
   case object InvalidNetworkIdentifier extends RosettaError(value = 4, message = "Invalid network identifier", retriable = false)
   case object InvalidPublicKey extends RosettaError(value = 5, message = "Invalid public key", retriable = false)
+  case object MalformedTransaction
+      extends RosettaError(
+        value = 9,
```

Comment:
Maybe these error numbers are enforced by rosetta?

Reply:
Rosetta only enforces this:

> Instead of utilizing HTTP status codes to describe node errors (which often do not have a good analog), rich errors are returned using this object. Both the code and message fields can be individually used to correctly identify an error. Implementations MUST use unique values for both fields.

So you can put any non negative integer there. I think @marcinwadon is right.

Reply:
Let me verify this and get back to you guys.

Reply:
`value` has been set to `5`, the next highest integer.

Reply:
actually to `6` @cngo-github